docs/user/orion.rst

```diff
@@ -0,0 +1,97 @@
+- A button to restart the integrated browser. If Orion gets stuck, restarting the browser might solve the issue without restarting IntelliJ.
+
+Problems
+^^^^^^^^
+
+Please report any issues on `Orion's GitHub repository <https://github.com/ls1intum/Orion>`_.
+
+Participation in Orion
+----------------------
+
+Students can participate in programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the exercise using the integrated browser.
+ 2. After starting the exercise, click |open-in-intellij-button|.
+ 3. After cloning, the exercise can be solved in IntelliJ.
+ 4. To submit the changes, click |submit-button|. This automatically commits, pushes the local changes, and displays the result of the automatic tests in IntelliJ.
+
+Gif showcasing the usage for students:
+
+  .. image:: https://raw.githubusercontent.com/ls1intum/Orion/master/.github/media/orion_workflow.gif
+            :align: center
+            :width: 100%
+
+Exercise Creation in Orion
+--------------------------
+
+Instructors can set up programming exercises via Orion by performing the following steps:
+
+ 1. The exercise needs to be created as described at the exercise creation of programming exercises, step 1 and 2.
+ 2. After the creation, navigate to the instructor exercise overview using the integrated browser
+ 3. Each programming exercise provides a button to edit the exercise in Orion |edit-in-intellij-button|. The button is rightmost in the table an might require scrolling. This downloads the template, solution and test repository of the exercise.
+ 4. Edit the repository files in IntelliJ.
+ 5. To submit the changes, click |submit-button|. This commits and pushes all local changes to their respective repository.
+ 6. The integrated browser displays the editor to edit the problem statement.
+ 7. To test the code locally, click |test-locally-button|, which copies the tests with the local template or solution (whichever was selected) into a new folder and execute them locally.
+
+Assessment in Orion
+-------------------
+
+Tutors can assess programming exercises via Orion by performing the following steps:
+
+ 1. Navigate to the assessment dashboard of the exercise using the integrated browser.
+ 2. Click |assess-in-orion-button|. This automatically downloads the automatic tests.
+ 3. After downloading or opening the exercise in IntelliJ, the submission overview is shown in the integrated browser. Each submission can be opened in Orion. To start a new submission, click |start-assessment-in-orion-button|. This downloads the submission files and overwrites the previous submission.
+ 4. The student's code is in the directory ``assignment``. The tests are in the directory ``tests``. Additionally, there are two directories ``template`` and ``studentSubmission`` that contain internal files and should be ignored.
```

Comment:
don't the template and studentSubmission directories contain the template and studentSubmission files? It's a bit misleading that the student's code is found in assignment if there is a directory called studentSubmission. I know that this is not part of this PR, but it becomes visible in this PR :D is there the possibility to prefix such internal dirs with `internal` ? Just an idea...maybe off topic

Reply:
The absolute truth is the studentRepository is - technically - not even required, since git should be able to track what the tutor changed and be able to retrieve the last submission without having to store a copy. But I was unable to get the git plugin to do this. I also tried to configure the folder in a way that IntelliJ hides it automatically. This also failed. But now that I think of it its probably possible to get Orion to treat the files from studentRepository equal to the ones from assignment, so the tutor can open either with the same result. I will try to do that and update the documentation afterwards.